components/FlexibleHeader/README.md

```diff
@@ -151,7 +151,7 @@ required init?(coder aDecoder: NSCoder) {
 - (id)initWithNibName:(NSString *)nibNameOrNil bundle:(NSBundle *)nibBundleOrNil {
   self = [super initWithNibName:nibNameOrNil bundle:nibBundleOrNil];
   if (self) {
-    _headerViewController = [MDCFlexibleHeaderViewController new];
+    _headerViewController = [[MDCFlexibleHeaderViewController alloc] init];
```

Comment:
nit: We should be calling designated initializers here and below.
[[MDCFlexibleHeaderViewController alloc] initWithNibName:nil bundle:nil];
Happy to discuss in another PR.

Opened #2041 to track.